runtime/ftplugin/solidity.vim

```diff
@@ -0,0 +1,7 @@
+" Vim filetype plugin file
+" Language: Solidity
+" Previous Maintainer: Cothi (jiungdev@gmail.com) 
+" Original author: tomlion (https://github.com/tomlion/vim-solidity)
```

Comment:
Simply using `Author:` here is more 'standard' and is recognised by the syntax highlighters.